mappings/net/minecraft/server/ServerConfigList.mapping

```diff
@@ -8,15 +8,19 @@ CLASS net/minecraft/class_1661 net/minecraft/server/ServerConfigList
 		ARG 1 file
 	METHOD method_6267 getNames ()[Ljava/lang/String;
 	METHOD method_6268 fromJson (Lcom/google/gson/JsonObject;)Lnet/minecraft/class_1660;
+		ARG 1 jsonObject
 	METHOD method_6269 toString (Ljava/lang/Object;)Ljava/lang/String;
 		ARG 1 profile
 	METHOD method_6270 add (Lnet/minecraft/class_1660;)V
+		ARG 1 configEntry
 	METHOD method_6271 setEnabled (Z)V
 		ARG 1 enabled
 	METHOD method_6272 isEnabled ()Z
 	METHOD method_6273 get (Ljava/lang/Object;)Lnet/minecraft/class_1660;
+		ARG 1 object
 	METHOD method_6274 getFile ()Ljava/io/File;
 	METHOD method_6275 remove (Ljava/lang/Object;)V
+		ARG 1 object
```

Comment:
object a terrible name

Reply:
for an Object?

Reply:
do you really think they parse an object there?

Reply:
yes

Reply:
they serialize it in fact